### PR TITLE
ci: add vab job compiling v graphical examples

### PR DESCRIPTION
**Pull request by Larpon**
This PR will add a minimal subset of the build matrix from `vab`'s CI.

I'm adding this to test if V's C output can be compiled successfully with the 4 Android SDK compilers.

I've chosen to test `vab`'s minimum and maximum supported Android API and Java version levels - since these can affect the Android SDK build/compile process.
I'm also leaving out AAB packaging since this requires downloads of external tools and would have to be cached - which is overkill for the scope of these tests.

(`vab`'s own CI has a more in-depth build matrix covering more scenarios)

### CODE REVIEW

File: .github/workflows/vab_ci.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+jobs:
+  vab-compiles-v-examples:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [macOS-10.15, ubuntu-18.04, ubuntu-20.04]
+        java-version: [8, 15]
+        android-api: [27, 30]
+        v-example: ['flappylearning', 'sokol/particles', 'sokol/drawing.v', 'sokol/freetype_raven.v', '2048', 'fireworks', 'tetris']
```
Comment (spytheman) [inline .github/workflows/vab_ci.yml:19]: Is it really necessary to test all 12 combinations on each commit/PR in the main V repository?
The output of the V compiler is the same afaik.

Comment (Larpon) [reply .github/workflows/vab_ci.yml:19]: I know it's some heavy stuff - my initial idea was to test V's C output without a matrix but then realised that both API level and platform can affect compilation results (Architectures are included though) and I'm planing on jni support for V UI and for better integration in general which then also factor in the Java version used. I don't really know what's too much and too little testing with an ecosystem like Android. I know CI juice is sparse - hence the subset of the colossus CI of `vab` itself 😄

I'm fine with flattening it out to start out with and then use the minimum supported versions on one host OS (Ubuntu is fastest)?

Comment (Larpon) [reply .github/workflows/vab_ci.yml:19]: ... don't know if we should include macOS?

Comment (spytheman) [reply .github/workflows/vab_ci.yml:19]: Unfortunately, github actions have limits in terms of time and number of jobs running at the same time that are global for the whole repository, and every additional job counts towards them => only one is best and imho, the ones in the main V repository, should only test that the C output is not obviously wrong, not that it works on every possible combination of java apis/oses.

The full tests can be done in vab's own repository.

Comment (spytheman) [reply .github/workflows/vab_ci.yml:19]: I think just Ubuntu 20.04 with one Java version will produce the most results (i.e. that ensuring that changes to the compiler itself will not cause it to break) with minimum overhead.

**Pull request merged**